cmd/webhook/main.go

```diff
@@ -19,7 +19,7 @@ package main
 import (
 	"flag"
 
-	"k8s.io/klog"
+	"k8s.io/klog/v2"
```

Comment:
I would expect that klog and glog can be removed from go.mod. Can you run "go mod tidy" and see if that cleans it up?

Reply:
I just ran ``go mod tidy`` but no new changes show up.

Reply:
Can you run `go mod vendor` and `go mod tidy` again? I would expect that the entries under go.sum and https://github.com/kubernetes-sigs/gcp-filestore-csi-driver/tree/master/vendor/github.com/golang/glog to be removed.

Reply:
The weird thing is that ```go mod tidy``` always try to add k8s.io/klog v1.0.0 back. :(

Reply:
It probably means that one of the depenencies is still using klog v1